content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+<div class="alert alert-info">
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+Profiling libraries are shipped within the following tracing language library. Refer to the dedicated language page to learn how to enable profile connections for your application:
```

Comment:
It sounds like node will ship the profiler in its own repo for now.

Reply:
Let's address this in another PR as for now only Java is available.

Reply:
Fwiw, go and python are expected to be in the next releases of their respective libs.